app/views/publishers/vacancies/edit_vacancy_sections/_edit_job_location.html.slim

```diff
@@ -1,12 +1,11 @@
-= render "publishers/vacancies/error_tag", attributes: (Publishers::JobListing::JobLocationForm.fields + Publishers::JobListing::SchoolsForm.fields)
-
 = render_divs_for_fields(Publishers::JobListing::JobLocationForm)
 = render_divs_for_fields(Publishers::JobListing::SchoolsForm)
 
 = render ReviewComponent.new id: "job_location" do |review|
   - review.heading title: t("publishers.vacancies.steps.job_location"),
                    text: t("buttons.change"),
                    href: organisation_job_build_path(@vacancy.id, :job_location)
+    = vacancy_review_section_tag(vacancy, :job_location, Publishers::JobListing::JobLocationForm)
```

Comment:
See the deleted line in this file. We need to somehow incorporate the SchoolsForm or the schools step, as well as the job location step. Perhaps the helpers should take an array of form fields, in the same way as the deleted line in this file, since they don't directly use the form class itself.

Reply:
I've made some changes to account for this. Take a look.